.github/workflows/tests.yaml

```diff
@@ -330,4 +330,4 @@ jobs:
             python -m pip install autopep8
       - name: Check diff
         run: |
-            find gapic tests -name "*.py" | xargs autopep8 --diff --exit-code
+            find gapic tests -name "*.py" -not -path 'tests/integration/goldens/*' | xargs autopep8 --diff --exit-code
```

Comment:
Can a job be added that checks the goldens?

Reply:
Done.